src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -12,6 +12,7 @@
 namespace Symfony\Component\Cache\Adapter;
 
 use Doctrine\DBAL\Connection;
+use InvalidArgumentException;
```

Comment:
I don't know the details in this case ... but the policy of Symfony is to never import the classes of the root namespace. So, instead of this, we usually just put `\InvalidArgumentException` whenever this class is used in the code.

Reply:
Oh nice! I'll change it!